lib/modules/apostrophe-dialog-box-layout-modifier/views/outerLayoutBase.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
```

Comment:
This will crush all improvements to `outerLayoutBase.html` provided by future versions of apostrophe core.. Adding the placeholder container should be done in a similar way to the SEO module https://github.com/apostrophecms/apostrophe-open-graph#use